Source/FieldSolver/FiniteDifferenceSolver/EvolveB.cpp

```diff
@@ -197,6 +211,223 @@ void FiniteDifferenceSolver::EvolveBCartesian (
     }
 }
 
+void FiniteDifferenceSolver::EvolveRhoCartesianECT (
+    std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& Efield,
+    std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& edge_lengths,
+    std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& face_areas,
+    std::array< std::unique_ptr<amrex::MultiFab>, 3 >& Rhofield, int lev ) {
```

Comment:
Can `lev` be `const`?
```suggestion
    std::array< std::unique_ptr<amrex::MultiFab>, 3 >& Rhofield, const int lev ) {
```